src/routes/users.ts

```diff
@@ -83,5 +83,19 @@ userRouter.put('/set-languages', getUserFromToken, async (req, res) => {
   return res.json(updatedUser);
 });
 
+userRouter.put('/update', getUserFromToken, async (req, res) => {
+  // const { user } = res.locals;
+  // console.log(user);
+  const { userId, userName, email } = req.body;
```

Comment:
The user id is never sent from the front end. That's what the token is for. You will have to use the `user.id` on `res.locals`.

Reply:
True, I tried doing that, it was giving me the wrong user.id though.